server/src/main/java/org/opensearch/cluster/routing/allocation/FileCacheThresholdSettings.java

```diff
@@ -51,6 +51,12 @@
  */
 public class FileCacheThresholdSettings {
 
+    public static final Setting<Boolean> CLUSTER_FILECACHE_ACTIVEUSAGE_THRESHOLD_ENABLED_SETTING = Setting.boolSetting(
```

Comment:
I remember we discussed that  limit > 100 sufficient here any reason for adding separate setting. If it was needed can we add tests I don't see any UT's or IT's covering it.